Dockerfile

```diff
@@ -1,24 +1,32 @@
-FROM node:12.22.12-alpine
-LABEL maintainer="-"
-
-ARG GITHUB_SHA_ARG
+ARG FUNCTION_DIR="/function"
+FROM node:12-buster as build-image
+ARG FUNCTION_DIR
 ARG PANDOC_VERSION=2.19.2
-
-COPY . /src
-
-WORKDIR /src
-
-# Install pandoc
+ARG GITHUB_SHA_ARG
+RUN apt-get update && \
+    apt-get install -y \
+    g++ \
+    make \
+    cmake \
+    unzip \
+    libcurl4-openssl-dev
+RUN mkdir -p ${FUNCTION_DIR}
+COPY . ${FUNCTION_DIR}
+WORKDIR ${FUNCTION_DIR}
 RUN wget https://github.com/jgm/pandoc/releases/download/$PANDOC_VERSION/pandoc-$PANDOC_VERSION-linux-amd64.tar.gz \
     && tar -xvzf pandoc-$PANDOC_VERSION-linux-amd64.tar.gz --strip-components 1 -C /usr/local \
     && rm -rf pandoc-$PANDOC_VERSION-linux-amd64.tar.gz
+RUN npm install aws-lambda-ric 
+RUN npm install
+RUN npm start
 
-# Install node deps
-RUN npm install --quiet --production
+FROM node:12-buster-slim
```

Comment:
curious if there is a reason for not using the AWS base image for nodejs is it just because you kept most things the same from when it was on heroku? 

https://docs.aws.amazon.com/lambda/latest/dg/nodejs-image.html#nodejs-image-base

Reply:
That and also we were running into some pandoc-related issues when using the AWS base image